src/main/java/gregtech/api/metatileentity/multiblock/MultiblockAbility.java

```diff
@@ -9,40 +9,41 @@
 import net.minecraftforge.fluids.IFluidTank;
 import net.minecraftforge.items.IItemHandlerModifiable;
 
-import java.util.ArrayList;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 @SuppressWarnings("InstantiationOfUtilityClass")
 public class MultiblockAbility<T> {
+    public static final Map<String, MultiblockAbility<?>> NAME_REGISTER = new HashMap<>();
+    public static final Map<MultiblockAbility<?>, List<MetaTileEntity>> REGISTER = new Object2ObjectOpenHashMap<>();
```

Comment:
These should probably be renamed to `NAME_REGISTRY` and `REGISTRY` respectively.